### PR TITLE
Ft test dev branch

### PR DESCRIPTION
**Pull request by Azutech**
#ft/test-branch, Implement unit test
PR Description Template
 #### Description #### Type of change
 Please select the relevant option - [ ] Bug fix (non-breaking changes which fixes an issue)
 - [X] New feature (non-breaking changes which adds functionality)
 - [ ] Breaking change (fix or feature that could cause existing functionality to not work as expected)
 - [ ] This change requires a documentation update
 - [X] Chore #### How Has This Been Tested?
 please describe the test that you ran to verify your changes 
 - [X] End to end
 - [ ] Integration #### Checklist: - [X] My code follows the style guidelines of this project
 - [X] I have linted my code prior to submission
 - [ ] I have made corresponding changes to the documentation
 - [X] My changes generates no new warnings
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: build/config/database.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+"use strict";
```
Comment (forestbaba) [inline build/config/database.js]: Please delete this guy from git commit

File: build/server.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+"use strict";
```
Comment (forestbaba) [inline build/server.js]: and this guy too

File: build/tests/helpers/reporter.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+"use strict";
```
Comment (forestbaba) [inline build/tests/helpers/reporter.js]: Same here

File: build/tests/serverSpec.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+"use strict";
```
Comment (forestbaba) [inline build/tests/serverSpec.js]: Here too

Comment (Azutech) [reply build/tests/serverSpec.js]: The pull request has been updated

File: build/tests/helpers/reporter.js (hunk 3 above)
Comment (Azutech) [reply build/tests/helpers/reporter.js]: I have resolved this too

File: build/server.js (hunk 2 above)
Comment (Azutech) [reply build/server.js]: I have resolved this too.

File: build/config/database.js (hunk 1 above)
Comment (Azutech) [reply build/config/database.js]: I have resolved this too.

**Pull request closed without merging**